packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -469,6 +469,31 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  factory CookieManager() {
+    if (_instance == null) {
+      final MethodChannel methodChannel =
+          const MethodChannel('plugins.flutter.io/cookie_manager');
+      _instance = CookieManager.private(methodChannel);
+    }
+    return _instance;
+  }
+
+  CookieManager.private(this._channel);
```

Comment:
the name private has no special meaning.
this should be named `_` to make sure it is really private.